lib/bootstrap/bundler.rb

```diff
@@ -91,7 +91,7 @@ def setup!(options = {})
     # @option options [Array] :without  Exclude gems that are part of the specified named group (default: [:development])
     # @return [String, Exception] the installation captured output and any raised exception or nil if none
     def invoke!(options = {})
-      options = {:max_tries => 10, :clean => false, :install => false, :update => false, :local => false,
+      options = {:silence_root_warning => false, :max_tries => 10, :clean => false, :install => false, :update => false, :local => false,
```

Comment:
This is where your diff effectively removes the default value, since our `options` becomes the result of merging the given options onto the hard-coded hash.

Reply:
🤦🏼 this doesn't remove a default. How did I read it that way?

Reply:
I've removed the variable, so it is set across the board anyway - I don't think there is a place that seeing the bundler error would be useful or actionable for users, so I'm not sure what value not silencing it would bring